packages/amplication-data-service-generator/src/server/static/src/auth/userData.decorator.ts

```diff
@@ -1,7 +1,7 @@
 import { createParamDecorator, ExecutionContext } from "@nestjs/common";
 import { GqlContextType, GqlExecutionContext } from "@nestjs/graphql";
 //@ts-ignore
```

Comment:
is it possible to avoid ts-ignore now ?

Reply:
no, because it is still not the correct path. but we need to fix the check on static files and then I think we will be able to remove it